hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/ViewFsOverloadScheme.md

```diff
@@ -55,44 +55,44 @@ Here `<scheme>` should be same as the uri-scheme configured in fs.defautFS. For
 
 **Example 1:**
 
-If users want some of their existing cluster (`hdfs://mycluster`) data to mount with hdfs(`hdfs://mycluster`) and other object store clusters(`o3fs://bucket1.volume1.omhost/`, `s3a://bucket1/`), the following example configurations can show how to add mount links.
+If users want some of their existing cluster (`hdfs://cluster`) data to mount with hdfs(`hdfs://cluster`) and other object store clusters(`o3fs://bucket1.volume1.omhost/`, `s3a://bucket1/`), the following example configurations can show how to add mount links.
 
 
 ```xml
 <property>
-  <name>fs.viewfs.mounttable.Cluster./user</name>
-  <value>hdfs://mycluster/user</value>
+  <name>fs.viewfs.mounttable.cluster./user</name>
```

Comment:
is link not required after cluster?
fs.viewfs.mountable.cluster.link./user

Reply:
It's a good catch @ayushtkn. I have corrected this in all places docs. ex: there are places javadoc etc missed this link before. 
Also took a liberty to correct few other doc issues like mount-table.versionNumber.xml. It should have '.' instead of -.